src/ontology/mondo-edit.obo

```diff
@@ -20511,9 +20511,9 @@ id: MONDO:0001385
 name: cortical blindness
 def: "Visual impairment due to visual cortex dysfunction." [NCIT:P378]
 synonym: "cerebral visual impairment" EXACT [https://orcid.org/0000-0002-3458-4839]
-synonym: "CVI" EXACT ABBREVIATION [https://orcid.org/0000-0002-3458-4839]
```

Comment:
this can be ignored, see explanation here: https://github.com/monarch-initiative/mondo/pull/7351#discussion_r1508025046